tests/tests_unit/test_graph/test_loaders/test_rdf2dms.py

```diff
@@ -0,0 +1,106 @@
+import pytest
+from cognite.client import data_modeling as dm
+from rdflib import Namespace
+from rdflib.term import URIRef
+
+from cognite.neat.graph.loaders import DMSLoader
+from cognite.neat.graph.stores import MemoryStore
+
+
+@pytest.fixture()
+def car_case() -> MemoryStore:
+    store = MemoryStore(namespace=Namespace("http://example.com/"), prefixes={"ex": "http://example.com/"})
+    store.init_graph()
+    store.add_triples(
+        [
+            (URIRef("ex:Car1"), URIRef("rdf:type"), URIRef("ex:Car")),
```

Comment:
For this example structure above works, as all your classes, properties, and instances are in the same namespace. In addition your classes and properties are using DMS compliant characters.

Reply:
Doing as agreed, assume 'load ready' store with all relevant in the 'neat' namespace